docs/sources/jinja2-templating/_index.md

```diff
@@ -201,7 +201,8 @@ Built-in functions:
 ### Functions added by Grafana OnCall
 
 - `time` - current time
-- `tojson_pretty` - JSON prettified
+- `tojson` - dumps a structure to JSON
```

Comment:
nit: technically this one is built-in and is not added by OnCall

Reply:
I guess still worth to be documented